bigtable/client/metadata_update_policy.cc

```diff
@@ -43,6 +43,18 @@ MetadataUpdatePolicy::MetadataUpdatePolicy(
       std::make_pair("x-goog-request-params", std::move(value));
 }
 
+MetadataUpdatePolicy::MetadataUpdatePolicy(
+    std::string resource_name, MetadataParamTypes metadata_param_type,
+    std::string snapshot_id, std::string cluster_id) {
```

Comment:
You receive `snapshot_id` and `cluster_id` by value, but only use them as const references.  Please change.